dynawo/3rdParty/CMakeLists.txt

```diff
@@ -509,7 +529,8 @@ else()
       INSTALL_COMMAND     ""
     )
   else()
-    ExternalProject_Add(boost
+    ExternalProject_Add(
+                          boost
```

Comment:
why newline ? useless

Reply:
That makes it readable. As in C++ we write
int foo(
      argument 1,
      argument 2,
) ...
I just follow that rule.  I roll that back anyway

Reply:
Please be aware that I'm not familiar with that coding style, it is not the one we use in C++, neither in Powsybl. I do my best. Do we have a written set of rules about it ? Or a tool like clang_format ?

Reply:
I wrote a "dg2cmake" utility that applies changes in respect of the coding style. Now rules should always be applied, unless some bug...

Reply:
coding style is not written but it is in full display in the file so not very dfficult to follow...